ClassesAndObjects/VideoStore/Movie.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace VideoStore
+{
+    class Movie
+    {
+        string title;
+        string studio;
+        string rating;
+
+        public Movie(string title, string studio, string rating = "PG")
+        {
+            this.title = title;
+            this.studio = studio;
+            this.rating = rating;
+        }
+
+        public Movie[] GetPG(Movie[] arrayMovies)
+        {
+            //                                PG tagged movies
+             return  arrayMovies.Where(movie => movie.rating == "PG").ToArray();
+ 
+            ;
+        }
+
+        public void BeingCheckedOut()
+        {
+            
+        }
+
+        public void BeingReturned()
+        {
+            
+        }
+
+        public void ReceivingRating(double rating)
+        {
+            
```

Comment:
Vai tad šiem nebūtu jābūt aizpildītiem?

Reply:
Būtu bijis, ja sapratu pareizi tie bija domāti 11uzdevumam, kam uztaisīju klasi video ar to pašu funkcionalitāti.
Labojumā noņēmu viņus nost, Movie.cs tagad sanāk risinājums tikai 4. uzdevumam bez saistības ar 11to
Zināšu nākamajos uzdevumos paskatīties vai kodam jādarbojas vairāku uzdevumu starpā.